content/en/docs/guides/debugging.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Debugging
+description: >-
+  Explains the debugging process of gRPC applications using [grpcdebug]
+---
+
+### Overview
+[grpcdebug] is a command line tool within the gRPC ecosystem designed to assist developers in debugging and troubleshooting gRPC services. grpcdebug fetches the internal states of the gRPC library from the application via gRPC protocol and provide a human-friendly UX to browse them. Currently, it supports [Channelz]/[Health] Checking/CSDS (aka. [admin services]). In other words, it can fetch statistics about how many RPCs has being sent or failed on a given gRPC channel, it can inspect address resolution results, it can dump the in-effective xDS configuration that directs the routing of RPCs.
```

Comment:
To me "in-effective" sounds too much like ineffective, maybe we want to just sya "active".